README.md

```diff
@@ -82,6 +82,45 @@ VAR=abc
 Any number of `--strict` flags can be provided.
 No more forgotten template overrides or missing env vars!
 
+### Master templates
```

Comment:
Let's call it `source`
It would be more clear. And `master` is a hot topic right now.